webdriver-spec.html

```diff
@@ -8402,6 +8390,80 @@
+function ab(a){return a.replace(/^[^\S\xa0]+|[^\S\xa0]+$/g,"")}
+function bb(a,c,b){if(R(a,"BR"))c.push("");else{var d=R(a,"TD"),e=T(a,"display"),f=!d&&!(0&lt;=ka(cb,e)),g=void 0!==a.previousElementSibling?a.previousElementSibling:Ea(a.previousSibling),g=g?T(g,"display"):"",r=T(a,"float")||T(a,"cssFloat")||T(a,"styleFloat");!f||"run-in"==g&&"none"==r||/^[\s\xa0]*$/.test(c[c.length-1]||"")||c.push("");var u=Ya(a),w=null,l=null;u&&(w=T(a,"white-space"),l=T(a,"text-transform"));C(a.childNodes,function(a){b(a,c,u,w,l)});a=c[c.length-1]||"";!d&&"table-cell"!=e||!a||fa(a)||
+(c[c.length-1]+=" ");f&&"run-in"!=e&&!/^[\s\xa0]*$/.test(a)&&c.push("")}}function db(a,c){bb(a,c,function(a,c,e,f,g){3==a.nodeType&&e?eb(a,c,f,g):R(a)&&db(a,c)})}var cb="inline inline-block inline-table none table-cell table-column table-column-group".split(" ");
+function eb(a,c,b,d){a=a.nodeValue.replace(/[\u200b\u200e\u200f]/g,"");a=a.replace(/(\r\n|\r|\n)/g,"\n");if("normal"==b||"nowrap"==b)a=a.replace(/\n/g," ");a="pre"==b||"pre-wrap"==b?a.replace(/[ \f\t\v\u2028\u2029]/g,"\u00a0"):a.replace(/[\ \f\t\v\u2028\u2029]+/g," ");"capitalize"==d?a=a.replace(/(^|\s)(\S)/g,function(a,b,c){return b+c.toUpperCase()}):"uppercase"==d?a=a.toUpperCase():"lowercase"==d&&(a=a.toLowerCase());b=c.pop()||"";fa(b)&&!a.lastIndexOf(" ",0)&&(a=a.substr(1));c.push(b+a)}
+function Xa(a){if(Ra){if("relative"==T(a,"position"))return 1;a=T(a,"filter");return(a=a.match(/^alpha\(opacity=(\d*)\)/)||a.match(/^progid:DXImageTransform.Microsoft.Alpha\(Opacity=(\d*)\)/))?Number(a[1])/100:1}return fb(a)}function fb(a){var c=1,b=T(a,"opacity");b&&(c=Number(b));(a=S(a))&&(c*=fb(a));return c}
+function W(a,c,b,d,e){if(3==a.nodeType&&b)eb(a,c,d,e);else if(R(a))if(R(a,"CONTENT"))C(a.getDistributedNodes(),function(a){W(a,c,b,d,e)});else if(R(a,"SHADOW")){for(;a.parentNode;)a=a.parentNode;if(a instanceof ShadowRoot&&a)for(a=a.olderShadowRoot;a;)C(a.childNodes,function(a){W(a,c,b,d,e)}),a=a.olderShadowRoot}else gb(a,c)}
+function gb(a,c){a.shadowRoot&&C(a.shadowRoot.childNodes,function(a){W(a,c,!0,null,null)});bb(a,c,function(a,c,e,f,g){var r=null;1==a.nodeType?r=a:3==a.nodeType&&(r=a);r&&r.getDestinationInsertionPoints&&0&lt;r.getDestinationInsertionPoints().length||W(a,c,e,f,g)})};function hb(a){var c=[];Sa?gb(a,c):db(a,c);var b=c;a=b.length;for(var c=Array(a),b=k(b)?b.split(""):b,d=0;d&lt;a;d++)d in b&&(c[d]=ab.call(void 0,b[d]));return ab(c.join("\n")).replace(/\xa0/g," ")}var X=["_"],Y=h;X[0]in Y||!Y.execScript||Y.execScript("var "+X[0]);for(var Z;X.length&&(Z=X.shift());)X.length||void 0===hb?Y[Z]?Y=Y[Z]:Y=Y[Z]={}:Y[Z]=hb;; return this._.apply(null,arguments);}.apply({navigator:typeof window!='undefined'?window.navigator:null,document:typeof window!='undefined'?window.document:null}, arguments);}
+
```

Comment:
Drop empty line and move `</pre>` tag up behind `}`. Since this is in a `<pre>`, the whitespace will be visible.